### DISCUSSION TITLE
Primary monitor

### DISCUSSION THREAD

**Discussion by ghnk** [i3/i3#5029] [Q&A] [answered]
I have 2 monitors for my PC. They are LG 29WK600-W. I'm using GNOME on Ubuntu 22.04 LTS, I assigned monitor #1 on the left hand side for opening supporting documents, on the other hand, monitor #2 on the right hand side is the primary display. It works so well and I can move the mouse icon from monitor #2 to the left monitor(#1). But on i3wm, by default, monitor #1 and monitor #2 are automatically assigned as workspace 1 and monitor 2 respectively and everything will start on monitor #1 as if it is the primary monitor.
I could not move the mouse icon to the left, from monitor #2 to monitor #1, I can only move from monitor #2 to the right out of screen and enter the left side of monitor #1. How can I make it work like in Ubuntu GNOME ?
![Screenshot from 2022-06-26 14-59-44](https://user-images.githubusercontent.com/47736274/175806088-e2df3f5c-a8f5-4a94-8e11-3b14a63f12be.png)

**Comment by orestisfl** [comment, marked as answer]
You can use arandr as a x11-generic graphical tool to configure your monitors. Persisting the configuration can be done with configuration files (https://wiki.archlinux.org/title/xrandr) or just repeating the commands on startup.
